plugins/wazo-snom/10.1.46.16/plugin-info

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.3.0",
+    "description": "Plugin for Snom D3x5, D717, D7x5 in version 10.1.46.16",
+    "description_fr": "Greffon pour Snom Snom D3x5, D717, D7x5 en version 10.1.46.16",
+    "capabilities": {
+        "Snom, D375, 10.1.46.16": {
+            "sip.lines": 12
```

Comment:
The format of this file has changed, see for example https://github.com/wazo-platform/wazo-provd-plugins/blob/master/plugins/xivo-aastra/3.3.1-SP4/plugin-info

Reply:
it's just on capabilites ? like "lines" instead of "sip.lines"?
`
   "lines": 9,
            "high_availability": true,
            "function_keys": 6,
            "expansion_modules": 0,
            "switchboard": true,
            "protocol": "sip"
`

Reply:
@ymolinet exactly

Reply:
updated